README-install.md

```diff
@@ -49,7 +49,16 @@ accept host verification, try this:
 
 Allow connections on the following ports/protocols:
   * icmp (for ping)
+  
+    Run:
+
+        # iptables -A INPUT -p icmp --icmp-type 8 -s 0/0 -d **public_hostname** -m state --state NEW,ESTABLISHED,RELATED -j ACCEPT
```

Comment:
This depends a great deal on how the machine is provisioned.  For example, all that is required for AWS is to use a security group which allows these ports.  Same with OpenStack.
In addition, using iptables like this may conflict with the openshift installation which uses either iptables or firewalld.

Did you need to run this in order to make this work?